sys/src/9/port/devmnt.c

```diff
@@ -963,7 +963,7 @@ mntrpcread(Mnt *mnt, Mntrpc *r)
 	l = &r->b;
 	*l = nil;
 	do {
-		b = qremove(mnt->q);
+		b = qget(mnt->q);
```

Comment:
I betcha this is my fault from when I split up the devmnt IO. It sure seems right. We'll see.

I would prefer it if you could split this one change out into its own PR. The other changes are not really related.